src/TextArea/TextArea.less

```diff
@@ -9,6 +9,9 @@
   background: @neutral_white;
   box-sizing: border-box;
   width: 100%;
+  min-height: 3.5rem;
+  padding-top: 0.125rem;
+  padding-bottom: 0.125rem;
 }
```

Comment:
I would use size vars whenever possible to make sure we stick to our design system guidelines: https://github.com/Clever/components/blob/828fb17feae2ab568f95e3c14088c463004a6b9c/src/less/sizing.less

And for things like padding top/bottom, you can use `.padding--y`: https://github.com/Clever/components/blob/828fb17feae2ab568f95e3c14088c463004a6b9c/src/less/spacing.less#L193